### ISSUE TITLE
Vulnerabilities found in dependencies

### ISSUE THREAD

**Issue by rc-collins-investedge** [knex/knex#4050]
# Environment

Knex version: 0.21.5
Database + version: sqlite3
OS: Windows

# Bug

We are using Knex 0.21.5, and when we run our application through our vulnerability analysis software, it reports several "High" vulnerabilities in the dependencies for Knex.  They seem to be dependencies on the `kind-of` dependency in the `liftoff` library.  

# Error Message

Here is a link to the CVE: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-20149

![image](https://user-images.githubusercontent.com/70852361/94347230-6934cc80-0000-11eb-8515-169b07a095b7.png)

Here is the dependency chain, showing several vulnerable versions (3.22, 5.1.0, and 4.00 all appear in the list).

```
PS C:\Users\rcollins\Code\git\rnd\cloud-next-app\landlord> npm lis kind-of
express-landlord@2.0.4 C:\Users\rcollins\Code\git\rnd\cloud-next-app\landlord
`-- knex@0.21.5
  `-- liftoff@3.1.0
    +-- findup-sync@3.0.0
    | `-- micromatch@3.1.10
    |   +-- braces@2.3.2
    |   | +-- fill-range@4.0.0
    |   | | `-- is-number@3.0.0
    |   | |   `-- kind-of@3.2.2
    |   | `-- snapdragon-node@2.1.1
    |   |   +-- define-property@1.0.0
    |   |   | `-- is-descriptor@1.0.2
    |   |   |   +-- is-accessor-descriptor@1.0.0
    |   |   |   | `-- kind-of@6.0.3  deduped
    |   |   |   +-- is-data-descriptor@1.0.0
    |   |   |   | `-- kind-of@6.0.3  deduped
    |   |   |   `-- kind-of@6.0.3  deduped
    |   |   `-- snapdragon-util@3.0.1
    |   |     `-- kind-of@3.2.2
    |   +-- define-property@2.0.2
    |   | `-- is-descriptor@1.0.2
    |   |   +-- is-accessor-descriptor@1.0.0
    |   |   | `-- kind-of@6.0.3  deduped
    |   |   +-- is-data-descriptor@1.0.0
    |   |   | `-- kind-of@6.0.3  deduped
    |   |   `-- kind-of@6.0.3  deduped
    |   +-- extglob@2.0.4
    |   | `-- define-property@1.0.0
    |   |   `-- is-descriptor@1.0.2
    |   |     +-- is-accessor-descriptor@1.0.0
    |   |     | `-- kind-of@6.0.3  deduped
    |   |     +-- is-data-descriptor@1.0.0
    |   |     | `-- kind-of@6.0.3  deduped
    |   |     `-- kind-of@6.0.3  deduped
    |   +-- kind-of@6.0.3
    |   +-- nanomatch@1.2.13
    |   | `-- kind-of@6.0.3  deduped
    |   `-- snapdragon@0.8.2
    |     +-- base@0.11.2
    |     | +-- cache-base@1.0.1
    |     | | +-- has-value@1.0.0
    |     | | | `-- has-values@1.0.0
    |     | | |   `-- kind-of@4.0.0
    |     | | `-- to-object-path@0.3.0
    |     | |   `-- kind-of@3.2.2
    |     | +-- class-utils@0.3.6
    |     | | `-- static-extend@0.1.2
    |     | |   `-- object-copy@0.1.0
    |     | |     `-- kind-of@3.2.2
    |     | `-- define-property@1.0.0
    |     |   `-- is-descriptor@1.0.2
    |     |     +-- is-accessor-descriptor@1.0.0
    |     |     | `-- kind-of@6.0.3  deduped
    |     |     +-- is-data-descriptor@1.0.0
    |     |     | `-- kind-of@6.0.3  deduped
    |     |     `-- kind-of@6.0.3  deduped
    |     `-- define-property@0.2.5
    |       `-- is-descriptor@0.1.6
    |         +-- is-accessor-descriptor@0.1.6
    |         | `-- kind-of@3.2.2
    |         +-- is-data-descriptor@0.1.4
    |         | `-- kind-of@3.2.2
    |         `-- kind-of@5.1.0
    `-- object.map@1.0.1
      `-- make-iterator@1.0.1
        `-- kind-of@6.0.3  deduped
```

**Comment by kibertoad** [comment]
The problem is that these are transitive dependencies that come from `liftoff` package which seems to be no longer actively maintained, unfortunately.

**Comment by kibertoad** [comment]
Related: https://github.com/js-cli/js-liftoff/issues/115

**Comment by kibertoad** [comment]
Resolved in master

**Comment by kibertoad** [comment]
Released in 0.95.0
